webpack/gutenbergBlocks.config.js

```diff
@@ -40,9 +40,9 @@ const {
 
 module.exports = ( mode ) => ( {
 	entry: {
-		// Reader Revenue Manager block.
-		'googlesitekit-reader-revenue-manager-block-editor':
-			'./assets/js/googlesitekit-reader-revenue-manager-block-editor.js',
+		// Reader Revenue Manager
+		'reader-revenue-manager/index':
+			'./blocks/reader-revenue-manager/index.js',
```

Comment:
Again, taking future changes into account let's update this:

```suggestion
		'reader-revenue-manager/block-editor-plugin/index':
			'./blocks/reader-revenue-manager/block-editor-plugin/index.js',
```